CHANGES.rst

```diff
@@ -66,6 +66,7 @@ Unreleased
 -   Choices which name and data are the same do not need to use tuples. (:pr:`526`)
 -   :class:`~validators.URL` validator now allows query parameters in the URL. (:pr:`523`, :pr:`524`).
 -   Updated French and Japanese translations. (:pr:`514`, :pr:`506`)
+-   Avoid to cache form errors. (:pr:`568`)

```

Comment:
I'd reword this to be more specific. "`form.errors` is not cached and will update if an error is appended to a field after access."